### DISCUSSION TITLE
Accordion pass throughs

### DISCUSSION THREAD

**Discussion by Ctrl-Zee** [primefaces/community#361] [PrimeReact] [answered]
I am trying to implement the example from the [accordian](https://primereact.org/accordion/) passthrough example. I have noticed that if I don't have the primeflex css imported I am unable to style the accordion tabs. 

Here is a link to the [StackBlitz](https://stackblitz.com/github/Ctrl-Zee/prime-accordion) example.

In the main.tsx file if you comment out `import "primeflex/primeflex.css";` the red background on the selected accordion tab disappears. 

Is primeflex a requirement for the accordion component or have I set something up incorrectly?

**Comment by habubey** [comment, marked as answer]
Hi, thanks for your question.
Of course, you don't need only Primeflex, and the reason you put it in the comment line is because bg-red-500 is a color of Primeflex and comes from there.
If you need to use your own styles or Tailwind you can do it with Passthrough.
Checkout the theming tab of https://primereact.org/accordion/  👍🏼

**Reply by Ctrl-Zee** [reply]
Thanks for the reply. I didn't realize that Primeflex came with colors defined. I think I had a misconfiguration with Tailwind in the project. You reply confirmed that for me.
